<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mFields_w</class>
 <widget class="QDialog" name="mFields_w">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Campos</string>
  </property>
  <widget class="QPushButton" name="mod_ok">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>259</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hecho</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>111</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Tipo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Longitud</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Key</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>161</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="mod_nombre"/>
    </item>
    <item>
     <widget class="QComboBox" name="mod_tipo"/>
    </item>
    <item>
     <widget class="QSpinBox" name="mod_length">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="mod_decimal">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mod_key">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
